packages/thegraph-data-access/src/types.ts

```diff
@@ -0,0 +1,9 @@
+import { StorageTypes } from '@requestnetwork/types';
+import { DataAccessBaseOptions } from '@requestnetwork/data-access';
+
+import { RequestInit } from 'graphql-request/dist/types.dom';
```

Comment:
What is this? Why is this imported from dist?

Reply:
because not exported from root :)
(nb: not new, moved)